man/tpm2-abrmd.8.in

```diff
@@ -70,6 +78,29 @@ behavior.
 .TP
 \fB\-v,\ \-\-version\fR
 Disply version string.
+.SH EXAMPLES
+.TP 3
+Execute daemon with default TCTI and options:
+.B tpm2-abrmd
+.TP
+Execute daemon with default TCTI and provided config string:
+.B tpm2-abrmd --tcti=":/dev/tpm0"
```

Comment:
Ahh We don't support this in tools, this harkens back to the "dead-code" comment.